docs/test_runners.md

```diff
@@ -0,0 +1,39 @@
+# Elección de `test runner`
+> Objetivo 4
+
+## Criterios
+1. Freshness (Valoraremos esto con `snyk`)
+2. Complejidad del `framework`.
+
+## Opciones
+> Tras leer en varios foros y teniendo en cuenta las características del modelo, las siguientes son las opciones que tendremos en consideración pese a que haya muchas alternativas más.
+
+### Mocha
+* Valoración de 89 puntos en [snyk](https://snyk.io/advisor/npm-package/mocha)
+* No incluye librería de aserciones, así que está abierto a combinarse con cualquiera.
+
+### Jest
+* Valoración de 95 puntos en [snyk](https://snyk.io/advisor/npm-package/jest)
+* Incluye su propia librería de aserciones, que no es demasiado extensa y cumple con los requisitos que se necesitan en este proyecto.
+
+### Jasmine
+* Valoración de 91 puntos en [snyk](https://snyk.io/advisor/npm-package/jasmine)
+* Su librería de aserciones es un tanto excasa, de todas formas queda descartado por el primer criterio.
+
+### Ava
+* Valoración de 98 puntos en [snyk](https://snyk.io/advisor/npm-package/ava)
+* Tiene su propia librería de aserciones `ava-assert`.
+
+## Comparación de librerías de aserciones
+
+De entre todas las librerías que he visto, sólo se siguen manteniendo `ava-assert`, con su último release hace 2 semanas y `Chai`. Ambas opciones son mucho más complejas (en cuanto a instrucciones) y completas, pero no incluyen nada que nos vaya a resultar útil que no incluya el framework de `Jest`.
```

Comment:
De veras, deberías hacer un pequeño esfuerzo por mirar las bibliotecas de aserciones (y test runners, en el sentido indicado en el guión) que hay.
El objetivo de aprendizaje es que tratéis de hacer una búsqueda de las herramientas existentes, no que listéis las cuatro más populares. Implica buscar usando el buscador de bibliotecas (o lo que sea) con cierta eficiencia, que es lo que es interesante, no que trasladéis lo que habéis leído en uno o dos artículos.
Hay muchas más bibliotecas de aserciones sólo en la primera página de ese buscador.